src/test/fixtures/upgrades/gecko-2.json

```diff
@@ -55,6 +55,32 @@
       "name": "examplebinary2"
     }
   ],
+  "counters": [
+    {
+      "name": "malloc",
+      "category": "Memory",
+      "description": "Amount of allocated memory",
+      "sample_groups": {
+        "id": 0,
+        "samples": {
+          "schema": {
+            "time": 0,
+            "number": 1,
+            "count": 2
+          },
+          "data": [
+            [0, 1, 1],
+            [2, 2, 2],
+            [4, 3, 3],
+            [5, 4, 4],
+            [8, 5, 5],
+            [9, 6, 6],
+            [10, 7, 7]
+          ]
+        }
+      }
+    }
+  ],
```

Comment:
I'm a bit relectant to add it here, because here this is a gecko profile version 13, and counters weren't implemented back then. They were added in version 14. So we're lying :/
(same problem for page information, likely, but let's say to not focus on this now).

My advice here would be to have a gecko-3.json with version 17 and a counter information.

Reply:
My problem is that with this we're testing something that doesn't exist. That's why I'm not comfortable about it.
But I won't block on this issue, so if you feel the other way I'm OK with that.